view-user/src/components/common/Button/Button.module.css

```
@@ -0,0 +1,53 @@
+.primary {
+  background-color: var(--background-color);
+  border: 1px solid black;
+  color: black;
+  text-align: center;
+  cursor: pointer;
+  position: absolute;
+  align-items: center;
+  display: flex;
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+
+}
+
+.l {
+  font-size: x-large;
+  padding: 0.2rem 0.2rem;
+
+}
+
+.m {
+  font-size: medium;
+  padding: 0.1rem 0.1rem;
+}
+
+.s {
+  font-size: x-small;
+}
+
+
+.circle {
+  border-radius: 2rem;
+}
+
+
+.square {
+  border-radius: 9px;
+}
+
+
+
+
+.icon {
+  background-color: var(--background-color);
+  color: black;
+  align-items: center;
+  padding: 4.8;
```

黒の色も#333333で統一したほうがいいかもです。
```suggestion
  color: #333333;
```